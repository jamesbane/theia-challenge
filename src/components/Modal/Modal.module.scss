@import "/src/styles/variables/common";
@import "/src/styles/mixins/breakpoints";
@import "/src/styles/animations";

.container {
  .head {
    background: #edeff4;
    min-height: 45px;
    padding: 17px 60px 17px 23px;
  }

  .title {
    font-family: $font-primary;
    font-size: 15px;
    font-weight: 500;
    font-stretch: normal;
    font-style: normal;
    line-height: normal;
    letter-spacing: normal;
    text-align: left;
    color: $font-dark;
    margin: 0;
  }

  .body {
    padding: 26px 23px 40px;

    &.noTitle {
      padding: 40px 23px 40px;
    }
  }

  :global {
    .ant-modal-content {
      border-radius: 15px;
      background: $bg-white;
      padding: 0;
      overflow: hidden;
    }

    .ant-modal-body {
      padding: 0;
    }

    .ant-modal-close-x {
      margin-right: 7px;
      margin-top: 5px;
    }
  }

}