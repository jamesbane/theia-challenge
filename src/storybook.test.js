import initStoryshots, {snapshotWithOptions} from '@storybook/addon-storyshots';
import { render } from '@testing-library/react';
import Modal from "./components/Modal";
initStoryshots({
    renderer: render,
    test: snapshotWithOptions({
        // @ts-ignore
        createNodeMock: (element) => {
            if (element.type === Modal) {
                return document.createElement('modal');
            }
        },
    }),
});