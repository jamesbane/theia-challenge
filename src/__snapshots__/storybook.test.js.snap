// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/Button Basic 1`] = `
<button
  className="button"
  onClick={[Function]}
>
  <span
    className="text"
  >
    Basic
  </span>
</button>
`;

exports[`Storyshots Components/Button Colors 1`] = `
<div>
  <button
    className="button buttonPrimary buttonBlue"
    onClick={[Function]}
  >
    <span
      className="text"
    >
      Primary & Blue
    </span>
  </button>
  <button
    className="button buttonPrimary buttonLight"
    onClick={[Function]}
  >
    <span
      className="text"
    >
      Primary & Light
    </span>
  </button>
  <button
    className="button buttonPrimary buttonGray"
    onClick={[Function]}
  >
    <span
      className="text"
    >
      Primary & Gray
    </span>
  </button>
  <button
    className="button buttonPrimary buttonGrayDark"
    onClick={[Function]}
  >
    <span
      className="text"
    >
      Primary & GrayDark
    </span>
  </button>
  <button
    className="button buttonPrimary buttonGradient"
    onClick={[Function]}
  >
    <span
      className="text"
    >
      Primary & Gradient
    </span>
  </button>
</div>
`;

exports[`Storyshots Components/Button Primary 1`] = `
<button
  className="button buttonPrimary"
  onClick={[Function]}
>
  <span
    className="text"
  >
    Primary
  </span>
</button>
`;

exports[`Storyshots Components/Button Size 1`] = `
<div>
  <button
    className="button buttonSmall"
    onClick={[Function]}
  >
    <span
      className="text"
    >
      Small
    </span>
  </button>
  <button
    className="button buttonCompact"
    onClick={[Function]}
  >
    <span
      className="text"
    >
      Compact
    </span>
  </button>
  <button
    className="button buttonPrimary"
    onClick={[Function]}
  >
    <span
      className="text"
    >
      Primary
    </span>
  </button>
</div>
`;

exports[`Storyshots Components/Button With Icon 1`] = `
<div>
  <button
    className="button buttonPrimary"
    onClick={[Function]}
  >
    <span
      className="text"
    >
      Button with Icon
    </span>
    <img
      alt="icon"
      className="icon"
      src="notify.png"
      style={
        Object {
          "width": 20,
        }
      }
    />
  </button>
  <button
    className="button buttonPrimary"
    onClick={[Function]}
  >
    <img
      alt="icon"
      className="icon iconAlignStart"
      src="notify.png"
      style={
        Object {
          "width": 20,
        }
      }
    />
    <span
      className="text"
    >
      Button with Icon
    </span>
  </button>
</div>
`;

exports[`Storyshots Components/Panel Basic 1`] = `
<div
  className="container"
>
  <div
    className="scroll"
    style={
      Object {
        "maxHeight": "none",
      }
    }
  >
    Panel Content
  </div>
</div>
`;
